api/resources/views/posts/create.blade.php

```diff
@@ -0,0 +1,32 @@
+@extends('layouts.app')
+
+@section('title', '| Create New Post')
+
+@section('content')
+<div class="container">
+    @include('errors')
+    @include('success')
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Create New Post') }}</div>
+                <div class="card-body">
+                    {!! Form::open(array('route' => 'posts.store')) !!}
```

Comment:
I don't really see a benefit in using the form builder instead of just making regular forms, but if it's better for you then that's fine by me.

Reply:
it looks fancier :D